code/core/src/telemetry/get-application-file-count.ts

```diff
@@ -0,0 +1,32 @@
+import { sep } from 'node:path';
+
+import { execCommandCountLines } from './exec-command-count-lines';
+import { runTelemetryOperation } from './run-telemetry-operation';
+
+// We are looking for files with the word "page" or "screen" somewhere in them with these exts
+const nameMatches = ['page', 'screen'];
+const extensions = ['js', 'jsx', 'ts', 'tsx'];
```

Comment:
`mjs` / `cjs`?

Reply:
Are `mjsx` and `cjsx` a thing?

Reply:
I guess the answer is kinda no: https://www.reddit.com/r/webdev/comments/15mern2/is_there_such_a_thing_as_cjsxmjsx_in_javascript/

And that might be a good reason to not include `mjs` and `cjs` in this script -- they are sort of guaranteed to be node files rather than browser files, right? Or is that an incorrect assumption?

Reply:
`mjs` can be browser files AFAIK